reader/src/test/java/org/kaazing/monitoring/reader/agrona/extension/CountersManagerExTest.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2007-2014 Kaazing Corporation. All rights reserved.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.kaazing.monitoring.reader.agrona.extension;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import org.jmock.Expectations;
+import org.jmock.Mockery;
+import org.jmock.lib.legacy.ClassImposteriser;
+import org.junit.Test;
+
+import uk.co.real_logic.agrona.concurrent.AtomicBuffer;
+
+public class CountersManagerExTest {
+
+    @Test
+    public void testGetLongValueForIdShouldReturnCounterValue() {
```

Comment:
There is no need for unit test methods to start with the word "test" since they are already marked with the @Test annotation, so this can be renamed to getLongValueForIdShouldReturnCounterValue. To speed things up I will make this change before merging (and a couple of similar cosmetic changes).